src/Analyzers/Analyzers/src/SymbolNames.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Analyzers
+{
+    internal static class SymbolNames
+    {
+        public static readonly string ConfigureServicesMethodPrefix = "Configure";
+
+        public static readonly string ConfigureServicesMethodSuffix = "Services";
+
+        public static readonly string ConfigureMethodPrefix = "Configure";
+
+        public static class IApplicationBuilder
```

Comment:
I use this pattern in a few other places like Razor where we need to keep track of type names and other constants related to types/methods. Defining a hierarchy like this makes it easy to complete in VS.